test/pet.test.js

```diff
@@ -4,7 +4,7 @@ const Pet = require('../src/pet');
   let petCat;
 
   beforeEach(() => {
-     petCat = new Pet();    
+     petCat = new Pet(name);    
```

Comment:
Nicely done. By creating this beforeEach object, you will save yourself from repeating the same lines of code in each test.